source/_docs/scripts/service-calls.markdown

```diff
@@ -77,6 +77,8 @@ entity_id: switch.ac
 
 {% endraw %}
 
+Please note that templates can not be used in most standard Lovelace cards (the exception being the Markdown card).
```

Comment:
This is a page about scripts, I'm not sure why we should add a warning about Lovelace here?